tools/cluster.py

```diff
@@ -329,7 +329,7 @@ def launch(cluster_name, slaves,
     raise CommandError('Failed to created cluster {} after failures'.format(cluster_name))
 
 
-def destroy(cluster_name, delete_groups=False, region=default_region):
+def destroy(cluster_name, delete_groups=False, region=default_region, wait_termination=False, wait_timeout=10):
```

Comment:
minor: would be good to indicate what time unit this time out is (my suggestion, wait_timeout_seconds)